source/tests-library/security.md

```diff
@@ -61,38 +50,23 @@ Vijil's tests check for such indirect prompt injection attacks that use a variet
 - Braille
 - Morse code
 
-```{eval-rst}
-.. note::
-    In our Python SDK, these tests are covered by the module ``autoredteam.tests.encoding``.
-```
-
 ## Glitch
 
 LLMs are susceptible to '[glitch tokens](https://www.lesswrong.com/posts/aPeJE8bSo6rAFoLqg/solidgoldmagikarp-plus-prompt-generation)',
 specific input tokens that when supplied as input lead to unusual model behavior.
 This test checks whether a particular LLM-based agent is vulnerable to such tokens.
 
-```{eval-rst}
-.. note::
-    In our Python SDK, this test is covered by the module ``autoredteam.tests.glitch``.
-```
 
 ## Goodside
 
 This group of tests implements prompt injection attacks [found](https://simonwillison.net/2022/Sep/12/prompt-injection/)
 by AI security engieer Riley Goodside.
 
-Vijil covers 4 Goodside attacks through its tests that are intended to make the agent
+Vijil covers Goodside attacks through its tests that are intended to make the agent:
```

Comment:
we're dropping goodside right?

Reply:
No, all goodside probes are included they are just split across diff dimensions now. Some are in security.

Reply:
ah got it